shared/logger/action-transformer.js

```diff
@@ -68,6 +68,7 @@ const actionTransformMap = {
   [Chat2Gen.metaNeedsUpdating]: fullOutput,
   [Chat2Gen.updateMoreToLoad]: fullOutput,
   [Chat2Gen.setConversationOffline]: fullOutput,
+  [ConfigGen.bootstrap]: fullOutput,
```

Comment:
What's in here. Safe to log?

Reply:
Yeap, just the single boolean `isReconnect`